wire-ios/Wire-iOS/Sources/UserInterface/StartUI/Common/EmptySearchResultsView.swift

```diff
@@ -82,37 +82,48 @@ final class EmptySearchResultsView: UIView {
     private let isSelfUserAdmin: Bool
     private let isFederationEnabled: Bool
 
-    private let stackView: UIStackView
+    /// Contains the `stackView`.
+    private let scrollView = UIScrollView()
+    private let stackView = UIStackView()
     private let iconView     = UIImageView()
     private let statusLabel  = DynamicFontLabel(fontSpec: .normalRegularFont,
                                                 color: LabelColors.textSettingsPasswordPlaceholder)
-    private let actionButton: LinkButton
+    private let actionButton = LinkButton()
     private let iconColor = LabelColors.textSettingsPasswordPlaceholder
 
     weak var delegate: EmptySearchResultsViewDelegate?
 
-    init(isSelfUserAdmin: Bool,
-         isFederationEnabled: Bool) {
+    init(
+        isSelfUserAdmin: Bool,
+        isFederationEnabled: Bool
+    ) {
         self.isSelfUserAdmin = isSelfUserAdmin
         self.isFederationEnabled = isFederationEnabled
-        stackView = UIStackView()
-        actionButton = LinkButton()
         super.init(frame: .zero)
 
+        [scrollView, stackView, iconView, statusLabel, actionButton].prepareForLayout()
+        [iconView, statusLabel, actionButton].forEach(stackView.addArrangedSubview)
+
+        addSubview(scrollView)
+
+        scrollView.leadingAnchor.constraint(equalTo: leadingAnchor).isActive = true
+        scrollView.topAnchor.constraint(equalTo: topAnchor).isActive = true
+        trailingAnchor.constraint(equalTo: scrollView.trailingAnchor).isActive = true
+        bottomAnchor.constraint(equalTo: scrollView.bottomAnchor).isActive = true
```

Comment:
Could you use the array representation to activate all at once? From the Apple documentation of `activate()`:
> The effect of this method is the same as setting the [isActive](doc://com.apple.documentation/documentation/uikit/nslayoutconstraint/1527000-isactive?language=swift) property of each constraint to true. Typically, using this method is more efficient than activating each constraint individually.
https://developer.apple.com/documentation/uikit/nslayoutconstraint/1526955-activate

Like:
```swift
NSLayoutConstraint. activate([
    scrollView.leadingAnchor.constraint(equalTo: leadingAnchor),
    scrollView.topAnchor.constraint(equalTo: topAnchor),
    trailingAnchor.constraint(equalTo: scrollView.trailingAnchor), 
    bottomAnchor.constraint(equalTo: scrollView.bottomAnchor)
])
```

Reply:
done